backend/tests/test_amd_gpu_detect.py

```
@@ -0,0 +1,89 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from utils.platform_detect import is_amd_gpu_windows
+
+
+class TestAmdGpuWindows:
+    """Unit tests for is_amd_gpu_windows with mocks."""
+
+    @patch("utils.platform_detect.platform.system", return_value="Linux")
+    def test_returns_false_on_linux(self, _mock_system):
```

Ruff (0.15.10)

[warning] Fixture `_mock_system` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)